src/pages/plugins/plugins.module.scss

```diff
@@ -31,6 +30,63 @@
   font-size: 1.1rem;
 }
 
+.pluginDeprecatedCard {
+  border-radius: var(--ifm-card-border-radius);
+  box-shadow: var(--ifm-global-shadow-lw);
+  overflow: hidden;
+  background-color: #ffdfdf;
+
+  .pluginCardHeader {
+    justify-content: flex-start;
+    margin: 16px;
+    margin-bottom: 0;
+    padding: 0;
+    padding-bottom: 12px!important;
+    border-bottom: solid 1px rgba(0,0,0,.1)
+  }
+
+  h3 {
```

Comment:
This will be removed when you use a badge

Reply:
h3 changed to badge and deleted